src/convertToObject.js

```diff
@@ -1,18 +1,20 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .map(style => style.trim())
+    .filter(style => style !== '');
+
+  const styleObject = styles.reduce((acc, style) => {
+    const [key, value] = style.split(':').map(str => str.trim());
+
+    acc[key] = value;
+
+    return acc;
+  }, {});
+
+  return styleObject;
```

Comment:
you can proceed with a chain of methods and avoid creating additional variables
```suggestion
    .filter(style => style !== '')
    .reduce((acc, style) => {
      const [key, value] = style.split(':').map(str => str.trim());

      acc[key] = value;

      return acc;
    }, {});

  return styles;
```